__tests__/public-api.test.js

```diff
@@ -192,6 +192,29 @@ describe('public api', () => {
       expect(timeout).toEqual(options.timeout)
     })
 
+    it('should call custom onOpen function if provided', () => {
+      const ChildWithAlert = withAlert(Child)
+      const tree = TestUtils.renderIntoDocument(
+        <Provider template={AlertTemplate}>
+          <ChildWithAlert />
+        </Provider>
+      )
+
+      // const childWithAlert = TestUtils.findRenderedComponentWithType(tree, ChildWithAlert)
```

Comment:
Everything looks good, just please remove this line of commented code and we can get this merged!